src/sync/app.hpp

```diff
@@ -269,6 +269,23 @@ class App {
      * Log out the given user if they are not already logged out.
      */
     void log_out(std::shared_ptr<SyncUser> user, std::function<void(Optional<AppError>)> completion_block) const;
+    
+    /// Switches the active user to the user with the specified id. The user must
+    /// exist in the list of all users who have logged into this application, and
+    /// the user must be currently logged in, otherwise this will return an
+    /// AppError.
+    ///
+    /// @param user_id A String specifying the desired `user_id`
+    /// @param completion_block will return a SyncUser on success, an AppError on failure
+    void switch_user(const std::string& user_id,
+                     std::function<void(std::shared_ptr<SyncUser>, Optional<AppError>)> completion_block) const;
+    
+    /// Logs out and removes the user with the provided id
```

Comment:
Would probably be helpful to specify that this is only a local operation and doesn't affect the server in any way.

Reply:
Makes sense, I've updated the doc to reflect that